src/content/quickstart/openshift/globalnet/_index.md

```diff
@@ -64,6 +64,12 @@ When the cluster deployment completes, directions for accessing your cluster, in
 Please note that  `oc`, `aws-cli`, `terraform`, and `wget` need to be installed before the `prep_for_subm.sh` script can be run.
 {{% /notice %}}
 
+{{% notice note %}}
+The script deploys an `m5n.large` EC2 instance type by default, optimized for improved network throughput and packet rate performance,
+for the Submariner gateway node. Please ensure that the AWS Region you deploy to support this instance type. Alternatively, you can
```

Comment:
```suggestion
for the Submariner gateway node. Please ensure that the AWS Region you deploy supports this instance type. Alternatively, you can
```

Reply:
@tpantelis we don't deploy *an* AWS Region, we deploy OpenShift *to* an AWS Region, for e.g us-east-2.